hawc/apps/summary/filterset.py

```diff
@@ -0,0 +1,147 @@
+        form.fields["type"].choices = self.get_type_choices()
+        return form
+
+    def pop_published(self, form):
+        # hide published filter and modify layout
+        if not self.perms["edit"]:
+            form.fields.pop("published")
+            form.grid_layout.rows[0].columns[0].width = 8
+            form.grid_layout.rows[0].columns[1].width = 4
+            del form.grid_layout.rows[0].columns[2]
+
+
+class DataPivotFilterSet(VisualFilterSet):
+    type = df.ChoiceFilter(
+        method="filter_evidence_type",
+        label="Visualization type",
+        help_text="Type of visualization to display",
+        empty_label="<All>",
+    )
+
+    class Meta(VisualFilterSet.Meta):
+        model = models.DataPivot
+
+    def filter_queryset(self, queryset):
+        return super().filter_queryset(queryset).select_related("datapivotquery", "datapivotupload")
+
+    def get_type_choices(self):
+        choice_options = (
+            models.DataPivot.objects.filter(assessment=self.assessment)
+            .values_list("datapivotquery__evidence_type", flat=True)
+            .distinct()
+        )
+        choice_options = map(lambda x: x if x is not None else 999, choice_options)
+        choices = []
+        for choice in sorted(set(choice_options)):
+            try:
+                choice = constants.StudyType(choice)
+                choices.append((choice.value, f"Data pivot ({choice.label})"))
+            except ValueError:
+                choices.append((999, "Data pivot (File Upload)"))
+        return choices
+
+    def filter_evidence_type(self, queryset, name, value):
+        value = int(value)
+        if value == 999:
+            return queryset.filter(datapivotupload__id__gt=0)
+        return queryset.filter(datapivotquery__evidence_type=value)
+
+
+class SummaryTableFilterSet(BaseFilterSet):
+    title = df.CharFilter(field_name="title", lookup_expr="icontains", label="Title")
+    type = df.ChoiceFilter(
+        field_name="table_type",
+        label="Table type",
+        help_text="Type of summary table to display",
+        empty_label="<All>",
+    )
+    published = df.ChoiceFilter(
+        choices=[(True, "Published only"), (False, "Unpublished only")],
+        label="Published",
+        help_text="Published status for HAWC summary tables",
+        empty_label="<All>",
+    )
+
+    class Meta:
+        model = models.SummaryTable
+        fields = ["title", "type", "published"]
+        grid_layout = {
+            "rows": [
+                {"columns": [{"width": 6}, {"width": 3}, {"width": 3}]},
+            ]
+        }
+
+    def filter_queryset(self, queryset):
+        queryset = super().filter_queryset(queryset)
+        query = Q(assessment=self.assessment)
+        if not self.perms["edit"]:
+            query &= Q(published=True)
+        return queryset.filter(query)
+
+    def create_form(self):
+        form = super().create_form()
+
+        if not self.perms["edit"]:
```

Comment:
changed this to hide, instead of disabling. This reduces the UI complexity for browsers, who don't need to see this information and may confuse them.